backend/counties/management/commands/load_county_data.py

```diff
@@ -1,98 +1,136 @@
-        with open ('counties/raw/ms-counties.json') as json_file:
-            data = json.load(json_file)
-            for feature in data.get('features'):
-                geo_dict = {
-                'type': 'FeatureCollection',
-                'features': []
-                }
-                geo_dict.get('features').append(feature)
-                print(feature.get('properties'))
-                county_fips = feature['properties']['CNTY_FIPS']
-                print(county_fips)
-                c = County.objects.get(pk=county_fips)
-                print(c.fips)
-                county_geo_json = CountyBorderGeoJSON(county=c, geojson=geo_dict)
-                county_geo_json.save()
-            print('Saved GeoJSON for all of the county borders.')
-
-
-        # Create a GeoJSON "County" object for the entire state
-        with open ('counties/raw/ms-counties.json') as json_file:
-            ms = County.objects.create(pk='000')
-            data = json.load(json_file)
-            ms_geo_json = CountyBorderGeoJSON(county=ms, geojson=data)
-            ms_geo_json.save()
-            print('Now you have a single County object that represents the entire state. It has a primary key of 000.')
-            print('It exsits so that you can query a signle GeoJSON object with various lines for the entire state.')
-            print('Instead of selecting all counties and then compiling the GeoJSON for the entire state')
-
-
-        # create voting precincts and the geojson for the precincts
-        with open ('counties/raw/ms-precincts-12.geojson') as json_file:
-            data = json.load(json_file)
-            for feature in data.get('features'):
-                print(feature.get('properties'))
-                county_fips = feature['properties']['COUNTYFP10']
-                if county_fips != '':
+            # Creates every county with the FIPS code
+            with open(input, 'r') as f:
+                csv_reader = reader(f)
+                counter = 0
+                header = ['state', 'state_fips', 'county_fips', 'county_name', 'fips_class']
+                counties = []
+                for row in csv_reader:
+                    row_dict = {}
+                    for index, column in enumerate(row):
+                        key = header[index]
+                        row_dict.update({key: column})
+                    counties.append(row_dict)
+                for county in counties:
+                    county_name = re.sub(r'^ County$', '', county['county_name'])
+                    county_name
+                    c = County(fips=county['county_fips'],
+                               name=county['county_name'].replace(' County', ''),
+                               state=county['state'],
+                               state_fips=county['state_fips'],
+                               fips_class=county['fips_class'])
+                    c.save()
```

Comment:
This works but can become pretty inefficient for large quantities of objects. If efficiency is a consideration (and your `Model.save()` method doesn't perform any important steps, such as parsing a required field from the instance data before creating the object),  you can build an array of objects, then [use `bulk_create`](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#bulk-create) to add them to the database.

[Here's an example from BGA Pensions](https://github.com/datamade/bga-pensions/blob/f431742baffc8f204e4f3a47e72f77c5e0458054/pensions/management/commands/import_data.py#L47-L64) that creates objects in batches of 1000.

FYI, this is not a required change, though you are free to make it for some or all of these loops if you find they take a long time to run.

Reply:
Cool, didn't know this feature. Thanks, I'll add this to my todo for improving this.